intelmq/bots/parsers/netlab_360/parser.py

```diff
@@ -21,18 +22,28 @@ def parse_line(self, line, report):
 
             if report['feed.url'] in Netlab360ParserBot.DGA_FEED:
                 event.add('source.fqdn', value[1])
-                event.add('time.source', value[3] + ' UTC')
+                # DGA Feed format is
+                # DGA family, Domian, Start and end of valid time(UTC)
+
+                event.add('time.source', value[2] + ' UTC')
```

Comment:
The last time this domain has been seen is in the fourth column, right?

I think this should be the latest time we know this event is true. @aaronkaplan?